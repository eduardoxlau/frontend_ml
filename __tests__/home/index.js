import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import App from "../../pages/index";

describe("App", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders without crashing", () => {
    const app = shallow(<App />);
    expect(app.find("container__Container").text()).toEqual(
      "Realiza una busqueda"
    );
  });
});
