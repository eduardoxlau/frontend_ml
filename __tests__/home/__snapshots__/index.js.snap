// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<container__Container
  className="title"
  display="flex"
  fontFamily="bold"
  fontSize={20}
  justifyContent="center"
  pt={90}
>
  Realiza una busqueda
</container__Container>
`;
